specs/brew/homebrew_tap_repository.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name: Tap a Homebrew tap repository from GitHub
+command: "brew tap {{tap_repository}}"
```

Comment:
Could we use `formula_repository` instead of `tap_repository` here? That also matches how brew references this argument: 
<img width="1138" alt="image" src="https://user-images.githubusercontent.com/4110292/164023797-045f327c-44e5-4e21-8432-9656a32f986a.png">